### ISSUE TITLE
Error when creating external table with Python API

### ISSUE THREAD

**Issue by djouallah** [datafuselabs/databend#11521]
### Search before asking

- [X] I had searched in the [issues](https://github.com/datafuselabs/databend/issues) and found no similar issues.

### Version

1.1.40

### What's Wrong?

trying to create an external table using Python API

### How to Reproduce?

```
ctx.sql(f"""
CREATE TABLE mytable (
  a INT
) 
external_location='s3://delta/databend/'
connection=(
  ACCESS_KEY_ID= {aws_access_key_id_secret}
  SECRET_ACCESS_KEY= {aws_secret_access_key_secret}
  endpoint_url= {endpoint_url_secret}
)
""")
```

got this error

```
RuntimeError                              Traceback (most recent call last)
[<ipython-input-21-556367454af0>](https://localhost:8080/#) in <cell line: 1>()
----> 1 ctx.sql(f"""
      2 CREATE TABLE mytable (
      3   a INT
      4 ) 
      5 external_location='s3://delta/databend/'

RuntimeError: Error: Code: 1005, Text = error: 
  --> SQL:6:12
  |
3 |   a INT
4 | ) 
5 | external_location='s3://delta/databend/'
6 | connection=(
  |            ^ expected <QuotedString>, <Ident>, <LiteralInteger>, <MySQLLiteralHex>, or <PGLiteralHex>
```

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

**Comment by sundy-li** [comment]
The documents are wrong.

https://github.com/datafuselabs/databend/blob/4665013d5de7a4551eb7d2fc1bd86d582faecd88/tests/suites/1_stateful/02_query/02_0001_create_table_with_external_location.sh

Example:

```
CREATE TABLE mytable (
  a INT
) 's3://delta/databend/' connection = (ACCESS_KEY_ID = 'xx' SECRET_ACCESS_KEY = 'xx'  endpoint_url = '');
```

**Comment by djouallah** [comment]
ok no error, but nothing was written to Cloudflare R2 ?

**Comment by wubx** [comment]
Look forward to using Databend in conjunction with R2, @djouallah  can you export a blog post?

**Comment by wubx** [comment]
Look forward to using Databend in conjunction with R2, @djouallah  can you export a blog post?

**Comment by BohuTANG** [comment]
The doc fixed in #11523
